libraries/AP_HAL_F4BY/AP_HAL_F4BY_Namespace.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace F4BY {
+	class F4BYScheduler;
+	class F4BYUARTDriver;
+	class F4BYStorage;
+	class F4BYRCInput;
+	class F4BYRCOutput;
+	class F4BYAnalogIn;
+	class F4BYAnalogSource;
+	class F4BYUtil;
+        class F4BYGPIO;
+        class F4BYDigitalSource;
+        class NSHShellStream;
+        class F4BYI2CDriver;
+        class F4BY_I2C;
+        class Semaphore;
+}
```

Comment:
We preparing new PR for F4BY HAL and intended to delete this file and to rename classes, but I checked it once more and just discovered there are a lot of places in master branch with a such kind of approach implemented. For example:

https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL/AP_HAL_Namespace.h
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_Empty/AP_HAL_Empty_Namespace.h
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_PX4/AP_HAL_PX4_Namespace.h
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_QURT/AP_HAL_QURT_Namespace.h
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_SITL/AP_HAL_SITL_Namespace.h
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_VRBRAIN/AP_HAL_VRBRAIN_Namespace.h

Are you sure we still need to refactor it?